core/include/cubos/core/ecs/query/data.hpp

```diff
@@ -75,8 +75,7 @@ namespace cubos::core::ecs
 
             // Initialize the fetchers with a templated functor which receives a sequence of indices (at compile time)
             // and initializes each fetcher for the argument term with the corresponding index.
-            auto initFetchers = [&]<std::size_t... Is>(std::index_sequence<Is...>)
-            {
+            auto initFetchers = [&]<std::size_t... Is>(std::index_sequence<Is...>) {
```

Comment:
Formatting error

Reply:
either my clang formatter is broken or idk.. formatting the file does not change anything